src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,74 @@
+package lotto.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lotto.domain.Budget;
+import lotto.domain.LottoMachine;
+import lotto.domain.dto.LottoResultDto;
+import lotto.domain.LottoNumber;
+import lotto.domain.dto.TicketDto;
+import lotto.domain.WinningNumbers;
+import lotto.view.LottoView;
+
+public class LottoController {
+    private final LottoMachine lottoMachine;
+    private final LottoView view;
+    private List<LottoNumber> lottoNumberPool;
+    private boolean holdFlag;
+
+    public LottoController() {
+        initNumberPool();
+        this.lottoMachine = new LottoMachine(lottoNumberPool);
+        this.view = new LottoView();
+        this.holdFlag = true;
+    }
+
+    public void start() {
+        List<TicketDto> tickets = null;
+        while (holdFlag) {
+            tickets = createTickets();
+        }
+        view.printTickets(tickets);
+
+        this.holdFlag = true;
+        LottoResultDto result = null;
+        while (holdFlag) {
+            result = getResult();
+        }
+        view.printLottoResult(result);
+    }
+
+    private List<TicketDto> createTickets() {
+        try {
+            int budget = view.getBudget();
+            List<TicketDto> tickets = lottoMachine.generateTickets(new Budget(budget));
```

Comment:
사소하지만 아래와 같이 변경하는게 조금 더 코드가 잘 읽히는 것 같아요. 😉
```suggestion
            Budget budget = new Budget(view.getBudget());
            List<TicketDto> tickets = lottoMachine.generateTickets(budget);
```

Reply:
반영했습니다!